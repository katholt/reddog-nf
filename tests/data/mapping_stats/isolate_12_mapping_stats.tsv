replicon	isolate	replicon_coverage	replicon_average_depth	replicon_reads_mapped	total_mapped_reads	total_reads	snps	snps_heterozygous	indels	pass_fail
contig_1	isolate_12	99.94	14.5	51.3998	35.0059	5102	0	0	0	f
contig_2	isolate_12	99.93	14.46	48.6002	35.0059	5102	0	0	0	p
