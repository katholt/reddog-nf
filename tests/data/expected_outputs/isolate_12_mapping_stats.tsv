replicon	isolate	replicon_coverage	replicon_average_depth	replicon_reads_mapped	total_mapped_reads	total_reads	snps	snps_heterozygous	indels	pass_fail
contig_1	isolate_12	99.94	14.5	17.9929	35.0059	5102	0	0	0	f
contig_2	isolate_12	99.93	14.46	17.0129	35.0059	5102	0	0	0	p
